helpers/common/00-cleanup

```diff
@@ -33,7 +33,7 @@ firewall_down()
 {
 	# Take down the firewall as some tests requires more than SSH port to be open (openMPI, NFSoRDMA, etc.)
 	local host=$1
-	tp $host 'iptables -P INPUT ACCEPT && iptables --flush'
+	tp $host 'systemctl stop SuSEfirewall2; true'
```

Comment:
Shouldn't we also support later SLE/OpenSUSE version by checking if should stop firewalld instead?